<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>491</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Configurar Atlas</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>451</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Mapa base:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nombre de Atlas:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QgsFieldComboBox" name="combo_atlas_field"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Metodo de Clasificacion</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Capa Atlas:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QComboBox" name="combo_methods">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QgsMapLayerComboBox" name="combo_atlas_layer"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="combo_basemap"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>491</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Configurar Dosis Preescrita</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="1">
      <widget class="QComboBox" name="combo_lbl_prees">
       <item>
        <property name="text">
         <string>Seleccionar...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>L/ha</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kg/ha</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Campo de Volumen:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string> Clases:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Unidades</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Capa Dosis Preescrita:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="spin_clases_prees">
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QgsFieldComboBox" name="combo_prees_field"/>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QgsMapLayerComboBox" name="combo_prees_layer"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>491</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Configurar Dosis Aplicada</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Capa Dosis Aplicada:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="combo_lbl_applied">
       <item>
        <property name="text">
         <string>Seleccionar...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>L/ha</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kg/ha</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Unidades</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Campo de Volumen:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="spin_clases_applied">
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string> Clases:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QgsMapLayerComboBox" name="combo_apl_layer"/>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QgsFieldComboBox" name="combo_apl_field"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>660</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Generar</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
